src/watchmaker/managers/platform.py

```diff
@@ -56,8 +56,8 @@ def __init__(self, system_params, *args, **kwargs):
         )
         self.system_params = system_params
         self.working_dir = None
-        args = args
-        kwargs = kwargs
+        PlatformManagerBase.args = args
```

Comment:
heh. `args = args` was done to make the linter happy in the first place. thinking about it now though, could probably be `self.args = args`....

Reply:
Okay. I added `ClassName.class_var` to the 6 offending cases but `self` is fine too if that's how it's being used. I'm guessing there's only one instance of `managers.PlatformManagerBase`, `managers.WorkersManagerBase`, and `workers.WorkerBase` so it's exactly equal.